frontend/src/components/SignUpform.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState  } from 'react'
+
+import { Article } from '../lib/FormStyle'
+import { Form } from '../lib/FormStyle'
+import { Input } from '../lib/FormStyle'
+
+
+
+export const SignUpform = () => {
+  
+  
+  const [signUpUser, setsignUpUser] = useState({
+      name: '',
+      email: '',
+      password: ''
+  })
+
+  const [error, setError] = useState('')
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    
```

Comment:
You don't clear the error anywhere, that's why it keeps showing even after success. Try to start the `handleSubmit()` with
```
event.preventDefault()
setError("")
```